gr-fec/python/fec/polar/channel_construction.py

```diff
@@ -27,8 +27,14 @@
 from channel_construction_bec import calculate_bec_channel_capacities
 from channel_construction_bec import design_snr_to_bec_eta
 from channel_construction_bec import bhattacharyya_bounds
-from channel_construction_awgn import tal_vardy_tpm_algorithm
 from helper_functions import *
+try:
+    from channel_construction_awgn import tal_vardy_tpm_algorithm
+except ImportError:
+    print "SciPy missing. Overwrite Tal-Vardy algorithm with BEC approximation"
```

Comment:
This print statement won't work with Py3k. Can you just add parens -- it's not quite the correct fix (you'd normally need to `from __future__ import print_function` as well) but it'll mean this particular hunk won't break Py3k.

Reply:
I just went through this file.
The print statements are mixed. Some are `print '...'` and some are `print('...')`.
I'd fix that file by consistently changing it to `print('...')`. Without `from __future__ import print_function` some prints might look weird, but I'd argue this is not a problem.

Reply:
Agree it's not a problem. We'll probably drop Py2k at some time anyway. However, if you want to change *all* the print statements in this file, I'd ask you to make 2 commits, one with the prints, and then the second with your fix. Thanks!

Reply:
The lines with print statements are: 34 (the one we discuss), 75 (already `print('...')`), 126 (needs to be fixed.), 135 (already `print('...')`).
All in all, I'd fix line 34 and 126 to follow Py3k `print_function` convention. That would be one new commit. I don't intend to squash my initial PR commit with this second one. Would that be the way to go?